pyproject.toml

```diff
@@ -0,0 +1,199 @@
+[project]
+# This is the name of your project. The first time you publish this
+# package, this name will be registered for you. It will determine how
+# users can install this project, e.g.:
+#
+# $ pip install sampleproject
+#
+# And where it will live on PyPI: https://pypi.org/project/sampleproject/
+#
+# There are some restrictions on what makes a valid project name
+# specification here:
+# https://packaging.python.org/specifications/core-metadata/#name
+name = "quantus"  # Required
+
+# Versions should comply with PEP 440:
+# https://www.python.org/dev/peps/pep-0440/
+#
+# For a discussion on single-sourcing the version, see
+# https://packaging.python.org/guides/single-sourcing-package-version/
+version = "0.4.1.dev1"  # Required
```

Comment:
Why do you write `0.4.1.dev1` instead of `0.4.1`?

Reply:
This is an in-development release version, as described here https://peps.python.org/pep-0440/#developmental-releases. I would like to first do an in-development release, and only after we are certain, nothing got broken, I'd update the tag to next minor one.

Reply:
> Why do you write `0.4.1.dev1` instead of `0.4.1`?

This is an in-development release version, as described here https://peps.python.org/pep-0440/#developmental-releases. I would like to first do an in-development release, and only after we are certain, nothing got broken, I'd update the tag to the next minor one.